daemon/cluster/convert/swarm.go

```diff
@@ -40,6 +49,8 @@ func SwarmFromGRPC(c swarmapi.Cluster) types.Swarm {
 				TrustRoot: string(c.RootCA.CACert),
 			},
 			RootRotationInProgress: c.RootCA.RootRotation != nil,
+			DefaultAddrPool:        defaultAddrPool,
+			SubnetSize:             int(c.SubnetSize),
```

Comment:
Wondering why this value has to be cast to an integer here; `c.SubnetSize` is an `uint32`, but `ClusterInfo.SubnetSize` is an `int`. Why?

Reply:
kept both uint32